ldapauthenticator/tests/test_ldapauthenticator.py

```diff
@@ -58,7 +58,26 @@ async def test_ldap_auth_blank_template(authenticator):
 
 async def test_ldap_auth_ssl(authenticator):
     authenticator.use_ssl = True
-    authenticator.server_port = 636
+
```

Comment:
Can you add a comment to these two tests saying we're not actually verifying SSL or TLS? for example if someone were to incorrectly modify the tls_strategy logic these tests would still pass.

Reply:
Good point! I pushed [df38bd1](https://github.com/jupyterhub/ldapauthenticator/pull/258/commits/df38bd1314b04f09390742c626b0414a6d8d5a42) (and unrelated df67ba7)

Reply:
after rebase: [f89de63](https://github.com/jupyterhub/ldapauthenticator/pull/258/commits/f89de634e841c04f273123522b90fb4adc355171) [d6178bc](https://github.com/jupyterhub/ldapauthenticator/pull/258/commits/d6178bc6a24f64b9afe5ae92a23cff1046d660eb)